programmers/팩토리얼.js

```diff
@@ -0,0 +1,12 @@
+const fs = require('fs');
+//const input = parseInt(fs.readFileSync('/dev/stdin').toString().trim());
+const input = 5;
+
+function factorial (N) {
+    if(N == 1 || N == 0) return 1;
+    else return N * factorial(N - 1);
```

Comment:
```suggestion
    if (N === 1 || N === 0) return 1;
    return N * factorial(N - 1);
```
저랑 비슷하게 푸셨네요.
문제 다 푸시고 컨벤션 한번 확인하는게 좋을것같아요.

input을 const로 선언해주셨는데 저는 let으로 선언해버렸네요 ㅜ 배워갑니다!